### ISSUE TITLE
Dissolve fisherman organization

### ISSUE THREAD

**Issue by jorgebucaran** [jorgebucaran/fisher#446]
## TL;DR

I'd like to dissolve the https://github.com/fisherman organization, please transfer your repositories to your own GitHub (or GitLab.com/BitBucket.org) account. 🙏 

## Background

I created the [fisherman](https://github.com/fisherman) organization because I thought that it would become the easiest way to contribute to the fish ecosystem, but what I've seen is that most people choose to host their packages in their own accounts and provide custom installation instructions for fisher, fundle, Oh My Fish! and so on. 

For this reason, I'd like to dissolve the organization. If you have any plugins or prompts (themes) of your own, please transfer them to your personal account and update the plugin installation instructions. Please note that fisherman will need the full URL _or_ the username/plugin_name (for GitHub users) in order to install your package.

### Instructions

1. Go to https://github.com/fisherman/{PACKAGE_NAME}/settings 
2. Scroll to the end, under Danger Zone look for Transfer ownership and click **Transfer**
3. Follow the instructions in the popup modal window
4. You did it! 🎉 

> **Note**: If you couldn't access the settings tab, you may not have owner permissions. Please let me know if this is the case and I'll make you an organization owner.

---

👋 cc @jethrokuan @Markcial @orefalo @FabioAntunes @igalic @daenney @gyakovlev

**Comment by jorgebucaran** [comment]
## TL;DR

I'd like to dissolve the https://github.com/fisherman organization, please transfer the repositories you own to your GitHub/GitLab account.

## Background

I created the [fisherman](https://github.com/fisherman) organization because I thought that it would become the easiest way to contribute to the fish ecosystem, but what I've seen is that most people prefer to distribute packages by their own means providing custom installation instructions for Fisherman, fundle, Oh My Fish! and so on. 

For this reason, I'd like to dissolve the organization. If you have any plugins or themes of your own, please transfer them to your personal account and update the plugin installation instructions. Please note that fisherman will need the full URL _or_ the username/plugin_name (for GitHub users) in order to install the package.

### Instructions

1. Go to https://github.com/fisherman/{PACKAGE_NAME}/settings 
2. Scroll to the end, under Danger Zone look for Transfer ownership and click **Transfer**
3. Follow the instructions in the popup modal window
4. You did it! 🎉 

> **Note**: If you couldn't access the settings tab, you may not have owner permissions. Please let me know if this is the case and I'll make you an organization owner.

---

cc @daenney 🙏

**Comment by sijad** [comment]
could you please transfer these two to my account:

https://github.com/fisherman/transfer
https://github.com/fisherman/gitignore

**Comment by jorgebucaran** [comment]
@sijad I can't transfer a repo from an organization to an account. It's a limitation of the GitHub organization system. I just made you an owner, though, so you should be able to do it yourself.

Please remember to update the installation instructions from:

<pre>
fisherman gitignore 
</pre>

to

```
fisherman sijad/gitignore
```

**Comment by daenney** [comment]
So, I maintained rbenv and pyenv. Problem is, I have no use for those plugins anymore as I've switched to using asdf to manage all my tools. I could transfer them to me, but it's unlikely I'll ever spend time on those plugins anymore. Not sure what to do in this case, essentially they're abandoned.

**Comment by jorgebucaran** [comment]
@daenney How about [archiving](https://help.github.com/articles/archiving-a-github-repository/) them after they are under your profile? Or would you like to avoid the transfer altogether? 🤔

**Comment by daenney** [comment]
Good idea. All done!

**Comment by jethrokuan** [comment]
done I think

**Comment by igalic** [comment]
done

**Comment by Markcial** [comment]
done 👷

**Comment by jorgebucaran** [comment]
Thank you @igalic & @Markcial!

👋 Ping @derhuerst @sotayamashita.

**Comment by kevinastone** [comment]
This dropped the `fisherman/tap` repo which was how I used to install fisherman from homebrew.  Was that the intention?  Should I migrate to the curlpipe?

**Comment by jorgebucaran** [comment]
@kevinastone There is no curlpipe <samp>curl url | fish</samp> and there is no install script. To install fisher we use <samp>curl -L https://git.io/fisher -o fisher.fish</samp> which downloads a file and that's all. And yes, please migrate to that. 🙏 

Keep in mind that even if I don't support installing fisher from homebrew, that doesn't mean someone else couldn't! I'm not against it, I just don't think it's worth it, at least not right now.

Why?

- not everyone has brew _or_ installing brew to install fisher is an avoidable extra step
- <samp>curl -L https://git.io/fisher -o fisher.fish</samp> will get you there on every mac
- <samp>wget https://git.io/fisher -O fisher.fish</samp> if you use wget instead
- because fish autoloads functions in `$fish_function_path`, e.g., ~/.config/fish/functions `fisher` is ready to run after dropping the file in the functions directory

**Comment by kevinastone** [comment]
Thanks.  That makes sense.  In my case, I can just version `fisher.fish` in my dotfiles repo with the rest of my fish config.

**Comment by jorgebucaran** [comment]
😅 Sorry for all the pings—promise this is the last one before I close the org (packages I don't own will be destroyed) @sotayamashita, @hutenosa @derhuerst.

**Comment by nesmyslny** [comment]
The eco prompt seems gone. Maybe it's related to transferring the repos?
The repo was https://github.com/fisherman/eco, but redirect now to https://github.com/fisherman/fin.

Same behavior as #300 two years ago.

I found this repo: https://github.com/dempfi/eco. Is this the new one?
Thanks for your help!

**Comment by jorgebucaran** [comment]
@nesmyslny The original one is gone, but you can use that fork if it works for you!

Most packages in the fisherman or (and oh my fish too) are prompts (themes). Most prompts do essentially the same, some more poorly than others. I dare to say that most people only want a nice prompt and don't care about the other features. However, I don't want to continue promoting the proliferation of superficial prompts. After V3 is out, I'll create a new prompt that can be heavily customized using env variables.

**Comment by hvdklauw** [comment]
I think the moving broke some stuff (maybe).

`fisher up` updates and then just hangs in a sleep loop.

Further inspection shows the following command just hanging and doing nothing:
`git-remote-https origin https://github.com/fisherman/docker-completion`

So I guess it got moved and I have to figure out where too myself and how to fix this?

**Comment by FabioAntunes** [comment]
@hvdklauw either got removed or the move was done badly. I moved mine if you go to  https://github.com/fisherman/nvm this will redirect to the new url automatically.

**Comment by jorgebucaran** [comment]
@hvdklauw Maybe try this https://github.com/barnybug-archive/docker-fish-completion?

**Comment by hvdklauw** [comment]
Thanks, I think indeed the move was done badly, no clue where else to report that as I don't know where it was moved to 😛

**Comment by jorgebucaran** [comment]
I just released fisher V3 and will be dissolving the organization over the next days (or weeks). 🎉 

## See Also

- [Release Notes](https://github.com/jorgebucaran/fisher/releases/tag/3.0.0)
- Migrating from V2 to V3 #450

**Comment by franciscolourenco** [comment]
@jorgebucaran "Organization members cannot transfer repositories" fisherman/done

<img width="338" alt="screen shot 2018-10-06 at 16 39 03" src="https://user-images.githubusercontent.com/208149/46572473-8c36c380-c986-11e8-84ee-52323fb127ce.png">

**Comment by franciscolourenco** [comment]
Any mechanism to assist migration to the new repositories? 

- What will happen when someone tries to upgrade `done` and it is no longer under fisherman? 
- What will happen when someone tries to upgrade `fisherman/done` and it is no longer under `fisherman/done`?

**Comment by jorgebucaran** [comment]
:wave: @franciscolourenco I made you an owner. You can transfer `done` to your own account now.

> What will happen when someone tries to upgrade done and it is no longer under fisherman?

An error message is shown when the user tries to install a package with an invalid package specifier such as `done`.

> What will happen when someone tries to upgrade fisherman/done and it is no longer under fisherman/done?

GitHub will automatically redirect the URL to correct resource.

**Comment by jorgebucaran** [comment]
@franciscolourenco Thank you!

**Comment by jorgebucaran** [comment]
Almost done—still waiting for:

- @kennethreitz: https://github.com/fisherman/pipenv by 
- @orefalo https://github.com/fisherman/free, https://github.com/fisherman/g2 and https://github.com/fisherman/javahome
- @sotayamashita https://github.com/fisherman/simple
- @hutenosa https://github.com/fisherman/autoenvstack
- @jethrokuan https://github.com/fisherman/koi

As admins you should be able to transfer your repositories to your account. If you don't want the repository, let me know and I'll dispose them.

**Comment by jorgebucaran** [comment]
Thank you, everyone, during the transition. The time has come. I'll delete the organization soon. Any packages still left in it will be deleted as well. 

Cheers. 👋

**Comment by jorgebucaran** [comment]
Almost done—still waiting for:

- @kennethreitz https://github.com/fisherman/pipenv  
- @orefalo https://github.com/fisherman/free, https://github.com/fisherman/g2 and https://github.com/fisherman/javahome
- @sotayamashita https://github.com/fisherman/simple
- @hutenosa https://github.com/fisherman/autoenvstack
- ~~@jethrokuan https://github.com/fisherman/koi~~

As admins, you should be able to transfer your repositories to your account. If you don't want the repository, let me know and I'll dispose of it for you.

**Comment by jorgebucaran** [comment]
I'll delete the organization over the next hours. 

Thank you for your cooperation during the transition and forgive me for the pings. There will be no more pings. 

Note that deleting the organization will delete all of its repositories. If you still haven't got around moving your repos, then I'm sorry.

**Comment by jorgebucaran** [comment]
I'll delete the organization over the next hours. 

Thank you for your cooperation during the transition and forgive me for the pings. There will be no more pings. 

Note that deleting the organization will delete all of its repositories. If you still haven't got around [moving your repos](https://github.com/jorgebucaran/fisher/issues/446#issuecomment-435594289), I'm sorry.

**Comment by jorgebucaran** [comment]
@nesmyslny IIRC you were using an eco fork. The package dependencies changed, so please send a PR to the maintainer or fork and update your fishfile:  

- ~~fisherman/segment~~ → [fishpkg/fish-segment](https://github.com/segment)
- ~~fisherman/git_util~~ → [fishpkg/fish-git-util](https://github.com/fish-git-util)
- ~~fisherman/humanize_duration~~ → [fishpkg/fish-humanize-duration](https://github.com/fish-humanize-duration)

🙏
